src/object.di

```diff
@@ -632,17 +632,17 @@ private void _destructRecurse(E, size_t n)(ref E[n] arr)
 
 private string _genFieldPostblit(S)()
 {
+    import core.internal.convert : itoa;
     import core.internal.traits : hasElaborateCopyConstructor;
-    import core.internal.traits : FieldNameTuple;
 
     string code;
-    foreach(fieldName; FieldNameTuple!S)
+    foreach(i, FieldType; typeof(S.init.tupleof))
     {
```

Comment:
Why not simply move that loop to `_postblitRecurse` and use `foreach (ref field; s.tupleof)`?
That would make the mixin superfluous.